docker-compose.yaml

```diff
@@ -173,6 +173,7 @@ services:
       - RAY_WORKER_GPUS_FRACTION
       - LUMIGATOR_API_CORS_ALLOWED_ORIGINS
       - MLFLOW_TRACKING_URI
+      - PYTHONPATH=/mzai/lumigator/backend:/mzai/lumigator/jobs
```

Comment:
Seems like this should go into the dockerfile? This is required for the thing to work

Reply:
I don't like hard coding these paths into the docker file. We may need more flexibility. Let's talk about it on Monday.

Reply:
The idea behind this is to expose the jobs paths so another path can be mounted from outside the container, for user specified jobs. I'd actually remove the baked-in env var in the Dockerfile, but I decided not to do it in this PR just in case. We may need to come up with an alternative entrypoint; I'd also rather use a shell script instead of running uv directly.

Reply:
OTOH, since such flexibility is not really important for this specific PR at the moment, we can make a new issue about it and keep it hardcoded for the moment.

Reply:
Removed